vnext/Microsoft.ReactNative/ReactApplication.h

```diff
@@ -19,19 +25,32 @@ namespace winrt::Microsoft::ReactNative::implementation {
 // It must be updated if the shape of generated ReactApplication_base is changed in future.
 // The only difference is that this class has no default constructor.
 template <typename D, typename... I>
-struct __declspec(empty_bases) NoDefaultCtorReactApplication_base
-    : implements<
-          D,
-          Microsoft::ReactNative::ReactApplication,
-          composable,
-          composing,
-          xaml::IApplicationOverrides,
-          xaml::IApplicationOverrides2,
-          I...>,
-      impl::require<D, xaml::IApplication, xaml::IApplication2, xaml::IApplication3>,
-      impl::base<D, xaml::Application>,
-      xaml::IApplicationOverridesT<D>,
-      xaml::IApplicationOverrides2T<D> {
+struct __declspec(empty_bases) NoDefaultCtorReactApplication_base : implements<
```

Comment:
why not just have NoDefaultCtorReactApplication_base defined twice with the appropriate args, wrapped inside a #ifdef clause. imo much more readable